stories/theme/ag/small/MobileListing.css

```diff
@@ -23,6 +23,22 @@
   margin: 8px 0 7px;
 }
 
+.MobileListing_NoMatchText {
+  position: absolute;
+  top: 50%;
+  max-width: 65%;
+  margin: 0 auto;
+  background: rgba(0, 0, 0, 0.5);
+  color: #FFFFFF;
+  border-radius: 5px;
+  padding: 0 6px;
+  text-align: center;
+  font-size: 12px;
+  font-weight: 700;
+  left: 50%;
```

Comment:
Not a big deal at all, but it'd be nice if we logically group attributes to make it easier to read. 
ex - position followed by top followed by left etc.